docs/document/content/reference/test/scaling-integration-test/_index.cn.md

```diff
@@ -0,0 +1,91 @@
++++
+title = "Scaling集成测试"
+weight = 4
++++
+
+## 测试目的
+
+验证Scaling自身功能和依赖模块的正确性。
+
+## 测试环境
+
+环境准备方式分为 Native 和 Docker，不论哪种环境，本地都需要预先安装Docker，
+
+- Native环境 用于本地调试，可以使用IDE的debug模式进行调试
```

Comment:
Every english word should be seperated by blank, e.g. `Native_`, `_IDE_`, `_debug_`.
And also other places.